script/deploy/devnet/M2_Deploy_From_Scratch.s.sol

```diff
@@ -87,28 +87,26 @@ contract Deployer_M2 is Script, Test {
     uint256 DELEGATION_INIT_PAUSED_STATUS;
     uint256 DELEGATION_WITHDRAWAL_DELAY_BLOCKS;
     uint256 EIGENPOD_MANAGER_INIT_PAUSED_STATUS;
-    uint256 EIGENPOD_MANAGER_MAX_PODS;
     uint256 DELAYED_WITHDRAWAL_ROUTER_INIT_PAUSED_STATUS;
 
     // one week in blocks -- 50400
     uint32 STRATEGY_MANAGER_INIT_WITHDRAWAL_DELAY_BLOCKS;
     uint32 DELAYED_WITHDRAWAL_ROUTER_INIT_WITHDRAWAL_DELAY_BLOCKS;
 
-    function run(string memory configFile) external {
+    function run(string memory configFileName) external {
         // read and log the chainID
         uint256 chainId = block.chainid;
         emit log_named_uint("You are deploying on ChainID", chainId);
 
         // READ JSON CONFIG DATA
-        deployConfigPath = string(bytes(string.concat("script/testing/", configFile)));
+        deployConfigPath = string(bytes(string.concat("script/configs/devnet/", configFileName)));
```

Comment:
Actually wondering if we shouldn't get rid of the `script/configs/devnet/` prefix and just let the user pass in an absolute path. This way any other avs repo that uses this script as a submodule could create its own deploy config file and pass it in, as opposed to having to go create a PR in eigenlayer-contracts to add a new config file, which doesn't make sense.